### ISSUE TITLE
Does not support Pylint's Parallel execution

### ISSUE THREAD

**Issue by jonykalavera** [edaniszewski/pylint-quotes#13]
False possitives will appear when pylint's jobs setting is != 1. 

https://pylint.readthedocs.io/en/latest/user_guide/run.html#parallel-execution

This happens when running against a project rather than a single file.

**Comment by edaniszewski** [comment]
What version of pylint are you using? I've tried reproducing this on some of my larger projects with different values set for the number of pylint jobs and haven't been able to get any false positives to show up. Are you using a non-default config for pylint-quotes? (all my testing was done using the default config)

**Comment by jonykalavera** [comment]
I'm using:
```
pylint==1.8.2
pylint-quotes==0.1.9
```
Let me try to put a simple example together.
